### DISCUSSION TITLE
with mozilla firefox the display is unresponsive and truncated

### DISCUSSION THREAD

**Discussion by bobwatcherx** [zauberzeug/nicegui#493] [Q&A] [answered]
I tried to be responsive by widening and narrowing the firefox window.  but that instead makes nicegui elements look truncated and not perfectly responsive.  but it works when i use google chrome browser

```py
from nicegui import ui

with ui.row().classes("w-full bg-blue-300 p-4"):
    ui.button("ssss").classes("bg-red-400")
    ui.button("xxxx").classes("bg-red-400")
    ui.button("xxxx").classes("bg-red-400")
    ui.button("xxxx").classes("bg-red-400")
    ui.button("xxxx").classes("bg-red-400")
    ui.button("xxxx").classes("bg-red-400")

ui.run()
```

**Comment by rodja** [comment, marked as answer]
It's working as expected on my Firefox (left) and Chrome (right):
![responsive](https://user-images.githubusercontent.com/131391/224463419-cde4b05f-e18b-453f-8fc4-95c123f43271.gif)
